nominal/nominal.py

```diff
@@ -362,7 +362,7 @@ def __print_human_readable_endpoint(self, endpoint):
         Print the Run datetime endpoints in a human-readable form
         """
         print("Run {} time:".format(endpoint))
-        unix_seconds = self._domain[endpoint]["SECONDS"] + self._domain[endpoint]["NANOS"] * 10e9
+        unix_seconds = self._domain[endpoint]["SECONDS"] + self._domain[endpoint]["NANOS"] // 1e9
```

Comment:
this was inverted, + an extra factor of 10 :)

Reply:
nit: I think we can ignore the nanos if we're converting to seconds as it's just intended to be the fractional part

Reply:
okay the word "endpoints" really confused me here - I thought this meant service endpoints, not "start" and "end" 😆 

suggested follow up to call this "bounds" instead of "domain" and/or just have `self.start_ts` and `self.end_ts` as fields